core/src/main/resources/error/error-classes.json

```diff
@@ -709,6 +709,11 @@
       }
     }
   },
+  "UNSUPPORTED_EMPTY_LOCATION" : {
+    "message" : [
+      "Unsupported empty location."
```

Comment:
empty location is a bit misleading. This is not a location with no files/directories, it's empty string. How about
```
EMPTY_STRING_AS_LOCATION: Can not reference a location with an empty string.
```